ratbagd/ratbagd.h

```diff
@@ -38,6 +38,10 @@
 #include "shared-macro.h"
 #include "shared-rbtree.h"
 
+#ifndef RATBAG_DBUS_INTF
+#define RATBAG_DBUS_INTF	"ratbag1"
+#endif
+
```

Comment:
I wonder if it'd be better to just `#define` all the object path and dbus prefixes in ratbagd.h and using them instead of the `RATBAG_DBUS_INTF` sprinkled everywhere